playbooks/roles/jumpbox/files/mysql_db_backup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
fyi, the first few commits in this pull request contain an unmodified version of this file from the rex pilot. i radically refactored it in order to a) split it into descriptive functions and b) reduce redundant code between mongo and mysql backup. i accomplished this by introducing shared_db_functions.sh